From 1d7df8402d00bfb5958c707d22672efae48d8cbb Mon Sep 17 00:00:00 2001
From: Salvo Giangreco <giangrecosalvo9@gmail.com>
Date: Sun, 31 Aug 2025 15:42:41 +0200
Subject: [PATCH] Allow custom CONNECTION_PERSONALIZATION value

UN1CA: this patch is not complete! Please check unica/patches/product_feature/customize.sh for more info.
---
 .../android/server/wifi/SemWifiInjector.smali | 68 ++++++++++---------
 1 file changed, 35 insertions(+), 33 deletions(-)

diff --git a/smali/org/apache/commons/compress/compressors/bzip2/BlockSort.smali b/smali/org/apache/commons/compress/compressors/bzip2/BlockSort.smali
--- a/smali/org/apache/commons/compress/compressors/bzip2/BlockSort.smali
+++ b/smali/org/apache/commons/compress/compressors/bzip2/BlockSort.smali
@@ -1,3 +1,3 @@
 .class public final Lorg/apache/commons/compress/compressors/bzip2/BlockSort;
-.super Ljava/lang/Object;
+.super Ljava/lang/Object;
 .source "qb/108127447 1eb5cabe977e6f074866ba43820b71a1f7453a78a4406bde13cd736320e5ac3a"
-- 
2.43.0
